client/js/widgets/widget.js

```diff
@@ -624,7 +624,11 @@ export class Widget extends StateManaged {
     const transformOrigin = getTransformOrigin(this.domElement);
     let positionCoord = this.coordParentFromCoordLocal(transformOrigin);
     const offset = getOffset(this.coordParentFromCoordLocal(localAnchor), positionCoord);
-    let corner = {x: coord.x + offset.x - transformOrigin.x, y: coord.y + offset.y - transformOrigin.y, z: this.get('z')};
+    let corner = {
+      x: Math.round(coord.x + offset.x - transformOrigin.x),
+      y: Math.round(coord.y + offset.y - transformOrigin.y),
+      z: this.get('z')
+  };
     return parent ?
         parent.coordGlobalFromCoordLocal(corner) : corner;
```

Comment:
You should probably round after the conversion to global coordinates. I.e.
```js
  if (parent)
    corner = parent.coordGlobalFromCoordLocal(corner);
  corner.x = Math.round(corner.x);
  corner.y = Math.round(corner.y);
  return corner;
```

This will ensure that we are consistent in rounding the global coordinates (for non-fixed parent widgets) rather than rounding local coordinates and then converting to global coordinates.

Reply:
Hopefully this is what you meant.